src/components/Icons/UploadFile/index.tsx

```diff
@@ -1,37 +1,72 @@
 import * as React from 'react';
 
-const UploadFile: React.FC<React.SVGProps<SVGSVGElement>> = ({ ...props }) => {
-  // eslint-disable-next-line no-console
-  console.warn('Icon.UploadFile is deprecated, please use Icon.Upload instead.');
```

Comment:
Isn't this icon deprecated anymore?

Reply:
This icon isn't the same as the Icon.Upload as it has a cloud shape. Resolving this comment.

Reply:
This icon has never been exported... it is only used within the upload component